### ISSUE TITLE
ChunkLoadError: Loading chunk node_modules_next_dist_client_dev_noop_js failed

### ISSUE THREAD

**Issue by paales** [graphcommerce-org/graphcommerce#1136]
<img width="992" alt="Screenshot 2021-09-28 at 12 06 52" src="https://user-images.githubusercontent.com/1244416/135068455-73367d47-299c-45cf-87a5-1e7e8f1c808e.png">
<img width="974" alt="Screenshot 2021-09-28 at 12 06 57" src="https://user-images.githubusercontent.com/1244416/135068460-c3f6afd0-a04e-498e-88dd-8fe99b50ffab.png">
<img width="971" alt="Screenshot 2021-09-28 at 12 07 04" src="https://user-images.githubusercontent.com/1244416/135068471-d7e1abb1-ea6b-41a2-9de0-304db02f191f.png">

Solution:
https://stackoverflow.com/questions/67652612/chunkloaderror-loading-chunk-node-modules-next-dist-client-dev-noop-js-failed

tldr; remove .next folder from the project and start again.
